### DISCUSSION TITLE
Paid name change model and proposal

### DISCUSSION THREAD

**Discussion by r4ntz** [ppy/osu#17438] [Ideas] [answered]
I understand the need to avoid name change abuse, but frankly paying more than ten dollars to change your name is a bit ridiculous. Peppy has had his username for years so he probably can't relate to indecisive users like myself which is understandable, but for users who have basically grown up with the game it's kind of sad to know that I'll be stuck with this cringe username unless I pay a hundred bucks. 

I'd like to suggest implementing a cooldown timer for username changes as a solution to the problems which can arise from name changes.

**Comment by peppy** [comment, marked as answer]
We can make special cases if you haven't changed your name in a long long time, please contact support at accounts@ppy.sh.

**Reply by SensouShin** [reply]
I think this should be an automated thing like: when you didn't change your name for X years then the free name change that is included with supporter is resets itself and you have to wait another X years for the next free
